mdc-103/starter/home.html

```diff
@@ -25,6 +25,7 @@ <h1 class="shrine-title">SHRINE</h1>
         <a class="mdc-list-item" href="#">
           Featured
         </a>
+        <li class="shrine-select-item-divider mdc-list-divider" role="separator"></li>
```

Comment:
This causes the starter to not match the previous codelab, as well as the screenshots to not match. Is there a reason we want to add the separator as part of the starter vs leaving it in the mdc-103 codelab?

Reply:
This step seems to be present in Step 4 of mdc-103 codelab. Closing this PR.

Thanks!